src/pfe/portal/modules/Project.js

```diff
@@ -718,7 +723,7 @@ async function isHcdSaved(pathToLoadTestDir) {
   const files = await fs.readdir(pathToLoadTestDir);
```

Comment:
This function should probably be renamed `findHcd` or `getHcdPath` to reflect what it does now.